### PR TITLE
add default GA plugins for handling page visibility, etc.

### PR DESCRIPTION
**Pull request by cvan**
this is mainly for the [`pageVisibilityTracker` plugin](https://github.com/googleanalytics/autotrack/blob/master/docs/plugins/page-visibility-tracker.md#overview); the other ones are useful too

(I borrowed this from https://github.com/MozillaReality/vr-mozilla-org/blob/master/static/js/ga.js)

### CODE REVIEW

File: Assets/WebGLTemplates/WebVR/lib/telemetry.js [hunk 1]
```diff
@@ -67,7 +67,13 @@ function setupAnalytics() {
       console.warn('Could not load Analytics.js script:', err);
       return;
     }
-  });  
+    injectScript('https://vr.mozilla.org/static/js/autotrack.js', function (err) {
```
Comment (delapuente) [inline Assets/WebGLTemplates/WebVR/lib/telemetry.js]: Do you mind to clarify what is in this file? Why do we need it?

Comment (cvan) [reply Assets/WebGLTemplates/WebVR/lib/telemetry.js]: it's useful for page visibility on mobile. it's a bit confusing, but the comments I was hoping would help. this doc help too: https://github.com/googleanalytics/autotrack/blob/master/docs/plugins/page-visibility-tracker.md#overview

I'll just close and handle this later. thanks though!

**Comment by delapuente** [comment]
Oh, I think I was ambiguous. I did not want to reject this. Actually, I like and agree on those changes. It is simply that I don't understand why do we need an additional script. In your link to GA documentation, I don't find the reference to that script. That's all. @cvan do you mind to reopen and provide an explanation of why do we need that link or remove it at all if not needed, please?

**Comment by cvan** [comment]
I'll resurrect this later. Oh, yeah, no offense taken. Thanks for explaining. I've filed MozillaReality/metrics#2 in the external repo to track. I'll bring this up later when we've pushed out Telemetry and this next milestone. Thanks, @delapuente!

**Pull request closed without merging**